ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -98,6 +99,7 @@ conformance_test(
         "@curl_nix//:bin/curl",
         "@grpcurl_nix//:bin/grpcurl",
         "@jq_dev_env//:jq",
+        "@jdk11_nix//:bin/java",
```

Comment:
on line 94 above we need `s/canton/canton_deploy.jar/` and ideally also add these two changes to the manual test below at lines 130 and 138

Reply:
good catch, fixed